safe-global/web-core
# fix: allow non-owner spending limit benficiaries to transact

## What it solves

Resolves #1561

## How this PR fixes it

Non-owner spending limit beneficiaries can now create and execute transactions according to their limits.

## How to test it

1. Add a non-owner as a spending limit beneficiary to a Safe.
2. Switch to the spending limit beneficiary
3. Observe that it is possible to create and execute transactions spending limit transactions with the associated token.

## Screenshots

![spending limit beneficiary](https://user-images.githubusercontent.com/20442784/213680242-ef242af3-833f-4901-8513-1a517b36c01f.gif)
